packages/gatsby-core-utils/src/__tests__/fetch-remote-file.js

```diff
@@ -279,11 +279,15 @@ describe(`fetch-remote-file`, () => {
   const cachePath = path.join(__dirname, `.cache-fetch`, `files`)
 
   beforeAll(async () => {
-    // Establish requests interception layer before all tests.
+    const originalFetch = global.fetch
```

Comment:
what are we doing here? 😄

Reply:
The problem was related to a conflict that occurs in the test environment when msw tries to overwrite global.fetch. In Node.js 20, global.fetch is a built-in function that is now treated as a read-only property. To solve this problem, we can temporarily remove global.fetch before calling server.listen() and restore it later.